nextjs-logs-manager/app/log_loading/actions.ts

```
@@ -0,0 +1,78 @@
+      try {
+        await prisma.log.create({
+          data: {
+            datetime: datetime,
+            typeId: type?.id,
+            service: serviceName,
+            message: message,
+          },
+        });
+      } catch (error) {
+        failed_loads += 1;
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)